plugins/main/public/utils/applications.ts

```diff
@@ -670,7 +670,7 @@ export const serverApi = {
   category: 'management',
   id: 'server-api',
```

Comment:
**nitpick**: change to `server-apis`.

Reply:
done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)